events/events_network.c

```diff
@@ -360,10 +363,14 @@ events_network_select(struct timeval * tv)
 	 * Convert timeout to an integer number of ms.  We round up in order
 	 * to avoid creating busy loops when 0 < ${tv} < 1 ms.
 	 */
-	if (tv->tv_sec >= INT_MAX / 1000)
-		timeout = INT_MAX;
-	else
-		timeout = tv->tv_sec * 1000 + (tv->tv_usec + 999) / 1000;
+	if (tv != NULL) {
```

Comment:
Can you put this backwards, so that it becomes if ... else if ... else ?  No point having unnecessary indentation.